setup.py

```diff
@@ -39,6 +39,8 @@
         'redlionfish',
         'magicgui',
         'pyqt5',
+        'mdocfile',
+        'joblib'
```

Comment:
Just adding a note here that this was for ctfsim improvements - no response required